### PR TITLE
Update config example

### PR DESCRIPTION
**Pull request by 1451678**

### CODE REVIEW

File: example.config.yml [hunk 1]
```diff
@@ -30,22 +30,23 @@ redmines:
       url: https://projects.example.net
       apikey:
     activities:
+      16: [AdministrationTechnisch, Adm, AT]
       16: [AdministrationTechnisch, Adm, AT]
```
Comment (BuJo) [inline example.config.yml:34]: There seems to be a duplication here?

File: example.config.yml [hunk 2]
```diff
@@ -30,22 +30,23 @@ redmines:
...
       10: [Analyse, Ana]
       26: [DailyWeekly, Daily, Dly, Wkly]
       8: [Design, Dsgn]
       17: [Dokumentation, Doku]
       22: [EinarbeitungGeben, EinG]
       28: [EinarbeitungEmpfangen, EinE]
-      27: [EntwicklungPair, EP]
-      9: [EntwicklungSolo, ES]
```
Comment (BuJo) [inline example.config.yml:41]: The new one here:

```yaml
81: [EntwicklungSolo, ES, Solo, Dev, EntwicklungPair, EP, Pair]
```

**Review by BuJo — CHANGES_REQUESTED** [review]

File: example.config.yml (hunk 1 above)
Comment (1451678) [reply example.config.yml]: Fixed.

File: example.config.yml (hunk 2 above)
Comment (1451678) [reply example.config.yml:41]: Done.

**Pull request merged**